performance/plugins.php

```diff
@@ -9,3 +9,13 @@
 add_action( 'amp_extract_image_dimensions_callbacks_registered', function() {
 	remove_filter( 'amp_extract_image_dimensions', array( 'AMP_Image_Dimension_Extractor', 'extract_from_attachment_metadata' ) );
 } );
+
+// CAP: By default, co-authors will query in a sort of compatibility mode.
+// Making the SQL queries work for posts with the old author style (as in the built-in post author
+// or by using the co-authors taxonomy.
```

Comment:
The English doesn't make sense here. 

It's also missing a `)`.

I'd also rather see it as a function DocBlock for `vip_coauthors_plus_should_query_post_author` below rather than just a random multiline-but-styled-as-a-single-line comment.

Reply:
It was a copy and paste from the description in the WPCOM snippet. I've re-written it now, @GaryJones.